crates/common_utils/src/types.rs

```diff
@@ -365,6 +365,11 @@ impl MinorUnit {
         self.0
     }
 
+    /// forms a new minor default unit i.e zero
+    pub fn zero() -> Self {
+        Self(0)
+    }
```

Comment:
You can have this as an associated constant instead
```rust
pub const ZERO: Self = Self(0);
```
Then to use it we can just do
```rust
let zero = MinorUnit::ZERO;
```
Just a nitpick, not a blocker.